ReferenceImplementation/CurrentStateProject.backend/src/CurrentStateProject/backend/Config.java

```diff
@@ -26,6 +27,11 @@
 
 	public final static HashMap<String, HashMap<String, String>> WORKFLOWELEMENT_EVENT_SUCCESSION = setAppWorkflowElementSuccession();
 	
+	public final static File UPLOAD_FILE_STORAGE_PATH = new File("c:/CurrentStateProject/");
```

Comment:
Reference implementation only. In actual usage, this is set in the Md2
model and generated according to the user's choosing
Am 03.03.2015 14:32 schrieb "Malte Möser" <notifications@github.com>:

> In
> ReferenceImplementation/CurrentStateProject.backend/src/CurrentStateProject/backend/Config.java
> <https://github.com/ps-md2/md2-projects/pull/17#discussion_r25686887>:
>
> > @@ -26,6 +27,11 @@
> >
> >  	public final static HashMap<String, HashMap<String, String>> WORKFLOWELEMENT_EVENT_SUCCESSION = setAppWorkflowElementSuccession();
> >  	
> > +	public final static File UPLOAD_FILE_STORAGE_PATH = new File("c:/CurrentStateProject/");
>
> Hardcoded path doesn't work on unix systems :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ps-md2/md2-projects/pull/17/files#r25686887>.
>